### ISSUE TITLE
got stuck running InferenceDataFromHDDM for a regression model

### ISSUE THREAD

**Issue by huashuoliu** [hcp4715/dockerHDDM#2]
Thank you for developing the docker! 
Recently I tried HDDM with my data and I got stuck when I run 'InferenceDataFromHDDM' for a regression model. The progress bar stops updating after reaching 105% and the netcdf file is not saved. 
Attached please find the output after I manually stopped the program.
[error_output.txt](https://github.com/hcp4715/dockerHDDM/files/10676266/error_output.txt)

**Comment by hcp4715** [comment]
Thanks for the shoutout! I will check the "InferenceDataFromHDDM" carefully. 
To reproduce your error, it's better also let me know the code of your model "ms1_1_tmp".

**Comment by huashuoliu** [comment]
Here's my code for defining the model "ms1_1_tmp".

[def_ms1_1.txt](https://github.com/hcp4715/dockerHDDM/files/10681498/def_ms1_1.txt)

**Comment by hcp4715** [comment]
Hi, @huashuoliu,

I tested the model you defined using example dataset (see[ here for the notebook](https://github.com/hcp4715/dockerHDDM_Guide/blob/master/temp/Notebook_for_Issues.ipynb)), but can not reproduce the error your reported. 

I noticed the `KeyboardInterrupt` in the error message you uploaded. If you didn't stop the process, it might be a "out of memory" problem, i.e., the capacity of your machine's memory is not enough for the temporary data generated during the process. It can be quit large if you run multiple models in-line. 

So I suggest you first rerun this model again while monitoring the status of memory consumption. Please let me know how it goes.

Best
Chuan-Peng

**Comment by huashuoliu** [comment]
I manually stopped the process after waiting for a long time without any updates in the output, so there's a 'KeyboardInterrupt' in the error message. 

I ran your notebook with both the example dataset and my data. It worked successfully. Since the only difference with my original code is the smaller number of samples and burn, I guess the problem might be relevant to some memory issue as you point out. I'll try rerunning my model later and update the result here. Many thanks for your timely reply!
